test/jtreg-ext/requires/VMProps.java

```diff
@@ -560,7 +559,7 @@ protected String hasLibgraal() {
      *
      * @return true if libgraal is used as JIT compiler.
      */
-    protected String isLibgraalEnabled() {
+    protected String isLibgraalJit() {
```

Comment:
I slightly prefer `isLibgraalJIT` as this acronym is (most) capitalized in the code base.

You should also rename `isLibgraalEnabled` to `isLibgraalJIT` in `test/lib/jdk/test/whitebox/code/Compiler.java` for consistency.

Reply:
Renamed both. I was being conservative by limiting scope of change to the relevant interface, and felt that the existing name was reasonable in the `Compiler` scope.